be-golang-plain/Jenkinsfile

```diff
@@ -0,0 +1,87 @@
+// build params
+def projectId = env.PROJECT_ID
+def componentId = env.COMPONENT_ID.toLowerCase()
+def packageName = env.PACKAGE_NAME
+def gitUrlHttp = env.GIT_URL_HTTP
+def gitUrlSsh = env.GIT_URL_SSH
```

Comment:
I don't see that used anywhere. Did I miss it? I added a short guide (https://github.com/opendevstack/ods-quickstarters/wiki/Experimenting-with-quickstarters) how to run these Jenkins pipelines WITHOUT a provisioning app (which is what I did today for testing) ... and in the example I did not need to pass a meaningful value for this env variable ...

Reply:
gitUrlHttp and componentId are used in every Quickstarter. gitUrlHttp is the URL to Bitbucket Repo where the checkin will be pushed. componentId is the "Project name".

packageName is only used in a few quickstarters. BeSpringBoot for example. Why the provisioning app creates a java packagename for quickstarters i dont know. 

gitUrlssh - i am getting rid of this one. There is no credential management (or at least i didnt find any) for ssh keys. Won't work with your AWS Setup anyway bcause ALB doesnt forward SSH.

Reply:
we should create a new ticket to rework the prov-app api / env variables passed to jenkins.

Reply:
@kiwo Yes I was talking ONLY about the `GIT_URL_SSH` env. For the others, I figured out there values and purpose :) I've created the issue here: #2.